### PR TITLE
Unit tests for TensorFlow importer

### PR DESCRIPTION
**Pull request by dkurt**
### This pullrequest changes
Several unit tests with generated inputs/outputs/networks from TensorFlow framework.
* Fix padding mode
* Element-wise addition, multiplication
* One more concatenation layer type
* Padding layer (very limited usage: only one side padding, no more than one padding per layer)
* Some activation layers (Abs, Sigmoid, TanH)
* Batch normalization

**Merge with extra**: https://github.com/opencv/opencv_extra/pull/358

### CODE REVIEW

**Comment by arrybn** [comment]
:+1:
